webapp/src/main/resources/public/js/ict/IctMetadataExtractor.js

```diff
@@ -37,10 +37,13 @@ class IctMetadataExtractor {
                         textUnit['textUnitName'] = spitTextUnitMetadata[2];
                         textUnit['locale'] = spitTextUnitMetadata[3];
                         textUnit['stack'] = spitTextUnitMetadata[4];
+                        textUnit['mtRequired'] = spitTextUnitMetadata[5];
+                        textUnit['mtLocale'] = spitTextUnitMetadata[6];
```

Comment:
how is "mtLocale" different from "locale"?

Reply:
I meant to update this, there was handling in the backend endpoint that converted pinterest locale to the mojito locale, if we were already in the mojito locale the map lookup failed and the original string got returned. I've updated the endpoint logic this morning to allow both pinterest & mojito locales so will remove the mtLocale field as it was a workaround to verify against the backend servers existing endpoint